docs/user/versions.rst

```diff
@@ -4,7 +4,8 @@ Versioned Documentation
 Read the Docs supports multiple versions of your repository.
 On initial import,
 we will create a ``latest`` version.
-This will point at the default branch for your VCS control: ``master``, ``default``, or ``trunk``.
+This will point at the default branch defined in your VCS control
+(by default, ``main`` on Git and ``default`` in Mercurial).
```

Comment:
We still default to master, what you probably mean is to mention something like "if you have a VCS account connected, we will default to the default branch from your VCS provider"

Reply:
You are right. However, I'm not sure how to explain this clearly because it depends on:

- the user has the VCS account connected
- the user has the remote repositories synced properly
- the user is importing the project via the wizard

Am I right?

I'm thinking of adding a note saying that when using the manual import it defaults to `master` on Git. However, this page will be re-written in the next sprints, so I don't want to deal with all of this refactor on this PR.

Reply:
I think we probably just write it assuming they have a RemoteRepo connected, and the others are an edge case?